android/src/main/java/com/agastya/androidinappupdates/AndroidInappUpdatesModule.java

```diff
@@ -63,8 +63,7 @@ protected void checkUpdate(final Promise promise, int appUpdateType, int clientV
 
         appUpdateInfoTask.addOnSuccessListener(appUpdateInfo -> {
             if (appUpdateInfo.updateAvailability() == UpdateAvailability.UPDATE_AVAILABLE
-                    && appUpdateInfo.clientVersionStalenessDays() != null
-                    && appUpdateInfo.clientVersionStalenessDays() >= clientVersionStalenessDays
+                    && (appUpdateInfo.clientVersionStalenessDays() == null ? clientVersionStalenessDays == 0 : appUpdateInfo.clientVersionStalenessDays() >= clientVersionStalenessDays)
```

Comment:
@renopp can you help to test and confirm the result?

Reply:
I tested my project using [Internal App Sharing](https://play.google.com/console/internal-app-sharing) and this fix works fine.

Reply:
@HorbachAndrii hey yes i think this should work fine, but as per kumparan policy any external contribution need to be reviewed by internal employee before merging it. Since I am no longger an employee at kumparan i can't merge this.

Reply:
I hope it doesn't take long